autogluon/task/image_classification/image_classification.py

```diff
@@ -1,43 +1,42 @@
-import os
-import copy
 import logging
+import os
+
 import mxnet as mx
-from mxnet import gluon, nd
-from ...core.optimizer import *
-from ...core.loss import *
-from ...core import *
-from ...searcher import *
-from ...scheduler import *
-from ...scheduler.resource import get_cpu_count, get_gpu_count
-from ..base import BaseTask
-from ..base.base_task import schedulers
-from ...utils import update_params
 
+from .classifier import Classifier
 from .dataset import get_dataset
+from .nets import *
 from .pipeline import train_image_classification
 from .utils import *
-from .nets import *
-from .classifier import Classifier
+from ..base import BaseTask
+from ..base.base_task import schedulers
+from ...core import *
+from ...core.loss import *
+from ...core.optimizer import *
+from ...scheduler.resource import get_cpu_count, get_gpu_count
+from ...utils import update_params
 
 __all__ = ['ImageClassification']
 
 logger = logging.getLogger(__name__)
 
+
 class ImageClassification(BaseTask):
     """AutoGluon Task for classifying images based on their content
     """
-    Classifier=Classifier
+    Classifier = Classifier
+
     @staticmethod
     def Dataset(*args, **kwargs):
         """Dataset for AutoGluon image classification tasks. 
            May either be a :class:`autogluon.task.image_classification.ImageFolderDataset`, :class:`autogluon.task.image_classification.RecordDataset`, 
-           or a popular dataset already built into AutoGluon ('mnist', 'cifar10', 'cifar100', 'imagenet').
+           or a popular dataset already built into AutoGluon ('mnist', 'fashionmnist', 'cifar10', 'cifar100', 'imagenet').
```

Comment:
@zhanghang1989 confirm if this is correct

Reply:
Thanks! This is correct.